modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/internal/servlet/taglib/BaseContainerTag.java

```diff
@@ -375,6 +375,26 @@ protected int processStartTag() throws Exception {
 		return EVAL_BODY_INCLUDE;
 	}
 
+	protected void renderIdAttribute(String _id) throws Exception {
+		JspWriter jspWriter = pageContext.getOut();
+
+		if (Validator.isNotNull(_id)) {
+			jspWriter.write(" id=\"");
+			jspWriter.write(_id);
+			jspWriter.write("\"");
+		}
+	}
+
+	protected void renderCssClassAttribute(String _cssClass) throws Exception {
```

Comment:
You need to use this method [in line 361](https://github.com/liferay-frontend/liferay-portal/pull/427/commits/135f0643d5da441dc8043bb58242999a70d77512#diff-2b4cdf2f6f59a9e1dcf930ab065f8d291606569ee9732953b52b52bff5b941eaR361) replacing those three lines (361 to 363)

Reply:
Also, replace `_id` with `getId()` everywhere in this fil. That's a change we're introducing in our cards PRs

Reply:
Literally everywhere? Even in `cleanUp` and in the last part of the class: `private String _id;`?

Reply:
No, everywhere where is being used, not assign/defined, which means everywhere but this two and obviously `setId`, `getId`, `setCssClass` and `getCssClass` methos.